src/vs/workbench/contrib/preferences/browser/preferences.contribution.ts

```diff
@@ -496,8 +496,8 @@ CommandsRegistry.registerCommand(OPEN_FOLDER_SETTINGS_COMMAND, function (accesso
 	return preferencesService.openFolderSettings(resource);
 });
 
-CommandsRegistry.registerCommand(OpenFolderSettingsAction.ID, serviceAccessor => {
-	serviceAccessor.get(IInstantiationService).createInstance(OpenFolderSettingsAction, OpenFolderSettingsAction.ID, OpenFolderSettingsAction.LABEL).run();
+CommandsRegistry.registerCommand(OpenFolderSettingsAction.ID, (serviceAccessor, args) => {
```

Comment:
I think we already support this for settings.

https://github.com/microsoft/vscode/blob/6eaf748366a48a5b6f9645cfa5e2257f51a027a6/src/vs/workbench/contrib/preferences/browser/preferences.contribution.ts#L230-L240

So not sure if you want to do this for all commands. @roblourens FYI

Reply:
Do you think people want this for the specific settings commands or is the main one sufficient @Vigilans?

Reply:
@roblourens If well documented in [built-in command reference](https://code.visualstudio.com/api/references/commands), I think the main one will cover most needs. If I'd say what more should be added, it would be better for `openWorkspaceCommand` to accept query argument, to allow extensions to prompt user to edit settings which will be saved in `settings.json`, instead of global user setting.